src/Wordpress/PluginActions.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace CF\WordPress;
+
+use CF\API\APIInterface;
+use CF\API\Request;
+use CF\Integration\DefaultIntegration;
+
+class PluginActions
+{
+    private $api;
+    private $config;
+    private $wordpressAPI;
+    private $dataStore;
+    private $logger;
+    private $request;
+
+    /**
+     * @param DefaultIntegration $defaultIntegration
+     * @param APIInterface       $api
+     * @param Request            $request
+     */
+    public function __construct(DefaultIntegration $defaultIntegration, APIInterface $api, Request $request)
+    {
+        $this->api = $api;
+        $this->config = $defaultIntegration->getConfig();
+        $this->wordpressAPI = $defaultIntegration->getIntegrationAPI();
+        $this->dataStore = $defaultIntegration->getDataStore();
+        $this->logger = $defaultIntegration->getLogger();
+        $this->request = $request;
+    }
+
+    /**
+     * GET /zones.
+     *
+     * @return mixed
+     */
+    public function loginWordPress()
+    {
+        $apiKey = $this->request->getBody()['apiKey'];
+        $email = $this->request->getBody()['email'];
+
+        // TODO: what should the exact response be?
+        $response = array(
```

Comment:
This is the correct response, remove the comment but move this response structure into CF\API\Plugin::createAPISuccessResponse($result) 
```
$response = array(
            'success' => 'true',
            'result' => $result
            );
```